index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Amel Portfolio</title>
+</head>
+
+<body>
+  <header class="header-class">
+    <nav class="nav-container">
+      <div class="flex-logo"><img src="./img/My-Logo.png" alt="logo"></div>
+      <div class="flex-menu"><img src="./img/Menu-button.png" alt="menu"></div>
+    </nav>
+  </header>
+
+  <div class="headline-section">
+    <div class="text">
+      <h1 class="big-text">I'm Amel Glad to see you!</h1>
+      <p class="small-text">I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+    </div>
+    <h2 class="blue-text">LET'S CONNECT</h2>
+    <div class="contact">
+      <a href="#"><img src="./img/twitter.png" alt="Twitter"></a>
+      <a href="#"><img src="./img/Linkedin.png" alt="linkedin"></a>
+      <a href="#"><img src="./img/medium.png" alt="Medium"></a>
+      <a href="#"><img src="./img/github.png" alt="Github"></a>
+      <a href="#"><img src="./img/angel-list.png" alt="Angel-list"></a>
+    </div>
```

Comment:
- [ ] It would be better if you use the `section` tag instead of `div` in order to make your code more Semantic 👏. To get more information about Semantic Html, see this  [link](https://www.freecodecamp.org/news/semantic-html5-elements/).

Reply:
Well solved